docs-2.0/nebula-exchange/use-exchange/ex-ug-import-from-clickhouse.md

```diff
@@ -91,7 +91,7 @@ After Exchange is compiled, copy the conf file `target/classes/application.conf`
   # Spark configuration
   spark: {
     app: {
-      name: Nebula Exchange {{exchange.release}}
+      name: NebulaGraph Exchange {{exchange.release}}
```

Comment:
need double check, will this config affect other settings? there are the same issues in the below topics